README.md

```diff
@@ -5,14 +5,19 @@
 
 This repository contains the materials for D-Lab [WorkshopName] workshop. Prior experience with [PriorWorkshopWithLink] is assumed.
 
-## Workshop Goals
+## Learning Objectives
 
 In this workshop, we provide [WorkshopGoal]. First, we’ll cover [Aim1]. Then, we’ll explore [Aim2]. 
 
+After this workshop, you will be able to [LearningObjective1]. You will [LearningObjective2] and [LearningObjective3].
```

Comment:
Can you convert the learning objectives and not-learning objectives both to lists?